### ISSUE TITLE
[BUG]: Definition of "Needs Review" inconsistent and missing actions to take to resolve occurrences reported

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#655]
### Project

a11y checker extension

### Browser

Chrome

### Operating system

Windows

### Description

Reported by users

Definition of "Needs review" is hard to find, but once found in two places is not consistent and does not have enough explanation to understand what to do about resolving the reported occurrences.

1. **Doc: User guide**: chrome-extension://lkcagbfjnkomcinoddgooolagloogehp/usingAC.html#categories
[needs review icon]  **Needs review** _- issues that may not be a violation. These need a manual review to identify whether there is an accessibility problem._

2. **Summary results card**:
**Needs review** [needs review icon]  [count]
**_Issues that may not be a violation; manual review is needed_**

[needs review icon] is also displayed in help message for individual "Learn more" panels, with no way to find definiton and no further guidance on how to eliminate or ignore the finding, but does include messages such as:
**_Confirm that ..._**
**_Verify there is a way to ..._**
**_Verify content is ..._**

### Steps to reproduce

1. Scan any website that results in "Needs review" occurrences

**Comment by aliunwala** [comment]
Triage:
Need too add to the design an easy way to find the definition of needs review. And at the same time make a easier way to get to the general purpose user guide/help

**Comment by MHoov** [comment]
FYI our Patterns team DevRev is looking at this as part of their project.
